### PR TITLE
feat: Do not use privileged Pod

### PR DESCRIPTION
**Pull request by scuzhanglei**

### CODE REVIEW

File: cmd/virt-prerunner/main.go [hunk 1]
```diff
@@ -30,8 +31,10 @@ import (
 func main() {
 	var vmData string
 	var receiveMigration bool
+	extraVFIOMemoryLockSize := resource.QuantityValue{Quantity: resource.MustParse("1Gi")}
```
Comment (fengye87) [inline cmd/virt-prerunner/main.go:34]: Why is this required?

Comment (scuzhanglei) [reply cmd/virt-prerunner/main.go:34]: For VFIO device passthouth, whole memrory lock is needed. and extra 1Gi is an empirical value from [libvirt](
 https://gitlab.com/libvirt/libvirt/-/blob/master/src/qemu/qemu_domain.c#L9350).

Comment (fengye87) [reply cmd/virt-prerunner/main.go:34]: So this is unrelated to this issue? It deserves another patch.

Comment (scuzhanglei) [reply cmd/virt-prerunner/main.go:34]: a `privileged` process can [lock any amount of memory](https://linux.die.net/man/2/prlimit). but an unprivileged process default can only lock a small amount of memory(`prlimit -l`). now we use `privileged=false` pod, we have to adjust this limit, and this is why we add `CAP_SYS_RESOURCE` capability.

**Review by fengye87 — COMMENTED** [review]
Have you tested this patch on both Kubernetes 1.24 and 1.25? KubeVirt's device plugin has some problem on 1.25

**Comment by scuzhanglei** [comment]
> Have you tested this patch on both Kubernetes 1.24 and 1.25? KubeVirt's device plugin has some problem on 1.25

I read the KubeVirt related issue and PR, we don't have that problem, and I just tested  on v1.25, it works.
